src/vs/base/browser/formattedTextRenderer.ts

```diff
@@ -94,6 +96,9 @@ function _renderFormattedText(element: Node, treeNode: IFormatParseTree, actionH
 		child = document.createElement('b');
 	} else if (treeNode.type === FormatType.Italics) {
 		child = document.createElement('i');
+	} else if (treeNode.type === FormatType.Preformatted && renderPreformattedSegements) {
+		child = document.createElement('pre');
+		(child as HTMLPreElement).style.display = 'inline';
```

Comment:
I believe that our markdown renderer outputs `<code>` blocks instead of `<pre>` for inline tags

Reply:
Makes sense, also saves me needing to manually override the `display`.